web/ui/module/codemirror-promql/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "codemirror-promql",
-  "version": "0.18.0",
+  "version": "0.19.0",
```

Comment:
you need to change the version in the react-app too.
Once you changed it run `npm install` in `web/ui` it will update the `package-lock.json`

Reply:
Ah right, done.